plugins/hosts/arch/host.rb

```diff
@@ -4,7 +4,7 @@ module VagrantPlugins
   module HostArch
     class Host < Vagrant.plugin("2", :host)
       def detect?(env)
-        File.exist?("/etc/arch-release")
+        File.exist?("/etc/arch-release") and not File.exist?("/etc/artix-release")
```

Comment:
```suggestion
        File.exist?("/etc/arch-release") && !File.exist?("/etc/artix-release")
```

Reply:
Thank you!

It's only right that the commit has the same style of the rest of the code. I was split between the two ways of writing this commit and I chose the one I thought was most readable (as a mainly Python dev).

Again, thanks for your attention and patience.